src/mtcp/mtcp_util.c

```diff
@@ -815,3 +815,28 @@ char *mtcp_getenv(const char *name, char **environ)
   }
   return NULL;
 }
+
```

Comment:
Something is weird in the history, here.  I remember writing this function.  But git says that this is a new function add in this commit.
Could it be that I wrote it for MANA, and you are copying it from MANA to here?  If so, could we add a small comment that the function was inherited from MANA?
Thanks.